File: ninchatsdk/src/main/java/com/ninchat/sdk/models/NinchatQueue.java
package com.ninchat.sdk.models;

/**
 * Created by Jussi Pekonen (jussi.pekonen@qvik.fi) on 28/08/2018.
 */
public final class NinchatQueue {

    public NinchatQueue(final String id, final String name, final long position) {
        this.id = id;
        this.name = name;
        this.position = position;
    }

    private String id;
    private String name;
    private long position = Long.MAX_VALUE;

    public String getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public long getPosition() {
        return position;
    }

    public void setPosition(long position) {
        this.position = position;
    }
}

File: ninchatsdk/src/main/java/com/ninchat/sdk/ninchatqueuelist/model/NinchatQueue.kt
package com.ninchat.sdk.ninchatqueuelist.model

/**
 * Created by Jussi Pekonen (jussi.pekonen@qvik.fi) on 28/08/2018.
 */
data class NinchatQueue(
    val id: String,
    var supportFiles: Boolean,
    var supportVideos: Boolean,
    val name: String?,
) {
    var position = Long.MAX_VALUE
    var isClosed = false
}